### PR TITLE
Make flag names consistent across projects.

### PR DESCRIPTION
**Pull request by juliusv**
@beorn7

### CODE REVIEW

File: web/web.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ import (
 
 // Commandline flags.
 var (
-	listenAddress  = flag.String("listenAddress", ":9090", "Address to listen on for web interface.")
-	useLocalAssets = flag.Bool("useLocalAssets", false, "Read assets/templates from file instead of binary.")
+	listenAddress  = flag.String("web.listen-address", ":9090", "Address to listen on for the web interface and API.")
```
Comment (brian-brazil) [inline web/web.go]: If we're going to be changing this flag name I think we should also ensure all the default ports are unique across projects while we're at it.

Comment (juliusv) [inline web/web.go]: Good point... I'll think of a list of ports and update the PRs.

**Comment by beorn7** [comment]
:+1:

File: web/web.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ import (
 
 // Commandline flags.
 var (
-	listenAddress  = flag.String("listenAddress", ":9090", "Address to listen on for web interface.")
-	useLocalAssets = flag.Bool("useLocalAssets", false, "Read assets/templates from file instead of binary.")
+	listenAddress  = flag.String("web.listen-address", ":9093", "Address to listen on for the web interface and API.")
```
Comment (juliusv) [inline web/web.go]: Changed this from 9092 -> 9093 to avoid collision with Apache Kafka.

**Pull request merged**